packages/jest-cli/src/lib/active_filters_message.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+import type {GlobalConfig} from 'types/Config';
+import chalk from 'chalk';
+
+const activeFilters = (
+  globalConfig: GlobalConfig,
+  delimiter: string = '\n',
+) => {
+  const {testNamePattern, testPathPattern} = globalConfig;
+  if (testNamePattern || testPathPattern) {
+    const filters = [
+      testPathPattern
+        ? chalk.dim('filename ') + chalk.yellow('/' + testPathPattern + '/')
+        : null,
+      testNamePattern
+        ? chalk.dim('test name ') + chalk.yellow('/' + testNamePattern + '/')
+        : null,
+    ]
+      .filter(f => !!f)
+      .join(', ');
+
+    const messages = ['\n' + chalk.bold('Active Filters: ') + filters];
```

Comment:
Why in an array? Did you mean to `concat` with `filters` instead of `join`ing on line 27?

Reply:
Yeah this doesn't seem to be needed, same as above just moved to from watch.js